source/dubregistry/web.d

```diff
@@ -595,6 +623,30 @@ class DubRegistryFullWebFrontend : DubRegistryWebFrontend {
 		redirect("/my_packages/"~_packname);
 	}
 
+	@auth @path("/my_packages/:packname/set_logo")
```

Comment:
FWIW this is usually mapped on `/logo` and only the http method is used to distinguish between get, post and delete.

Reply:
well all other endpoints are like this too

Reply:
Yeah, let's keep that for another day. The current scheme is just used to avoid HTTP method override hacks to account for the form based nature of the interface.